tests/Dummy/TestQueueManager.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Spatie\FailedJobMonitor\Test\Dummy;
+
+use Illuminate\Contracts\Queue\Job;
+use Illuminate\Queue\Jobs\SyncJob;
+
+class TestQueueManager
+{
+    protected $container;
+
+    public function __construct($container)
+    {
+        $this->container = $container;
+    }
+
+    protected function createPayload($job, $data = '', $queue = null) : string
+    {
+        if (is_object($job)) {
+            $payload = json_encode([
+                'job' => 'Illuminate\Queue\CallQueuedHandler@call',
+                'data' => [
+                    'commandName' => get_class($job),
+                    'command' => serialize(clone $job),
+                ],
+            ]);
+        } else {
+            $payload = json_encode($this->createPlainPayload($job, $data));
+        }
+
+        if (JSON_ERROR_NONE !== json_last_error()) {
```

Comment:
If fact createPayload took it from laravel/framework :D https://github.com/laravel/framework/blob/5.3/src/Illuminate/Queue/Queue.php#L79 I will rewrite it for you.